### DISCUSSION TITLE
Please stop re-enabling analytics at each update.

### DISCUSSION THREAD

**Discussion by FlorentLM** [adrianboguszewski/debug-image-viewer#7] [General]
That's a shit move. The only reason why I haven't subscribed for the premium version is this

**Comment by adrianboguszewski** [comment]
@FlorentLM I'm sorry to hear that. It's not designed to work like you describe. The settings are persisted on disk, so they shouldn't change between updates. If they do, something is wrong (a bug?). 
However, as they are local (saved to your SSD), you must turn off this on every computer. Is it your case?

**Reply by adrianboguszewski** [reply]
Ok. I found the problem. The settings implementation has changed to support different values for different projects and this reset all settings. Sorry for that 😐
